### ISSUE TITLE
:pact.verifier.publishResults not publishing results for leiningen project 

### ISSUE THREAD

**Issue by sonalchandani** [DiUS/pact-jvm#1005]
We tried setting the :pact.verifier.publishResults this way  `lein with-profile pact pact-verify :pact.verifier.publishResults="true" :pact.provider.version="provider-version"` but it isn't publishing the test verification results back to pact broker.

**Comment by aplsup** [comment]
Hi, is the "provider-version" you are posting the exact same one you are trying to send to the broker?
I think the broker only accepts _semantic versioned provider versions_ .
If the task is actually performing some http requests that could be the reason

**Comment by sonalchandani** [comment]
Yes we are sending a _semantic versioned provider version_. We posted this provider-version just for reference as example.

**Comment by uglyog** [comment]
I've just committed a change (d30509c) that will allow you to use an environment variable for the value
